modules/nw-osp-pod-connections-ipv6.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/network-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="nw-osp-pod-connections-ipv6_{context}"]
+= Enabling IPv6 connectivity to pods on {rh-openstack}
+
+To enable IPv6 connectivity between pods that have additional networks that are on different nodes, disable port security for the IPv6 port of the server. Disabling port security prevents the creation of allowed address pairs with IP and MAC addresses on the IPv6 ports of servers whenever a new pod is created.
```

Comment:
Actually, disabling port security makes it unnecessary to create allowed address pairs (which is difficult to apply, since MAC and IP address might only be known after pod creation).

Reply:
```suggestion
To enable IPv6 connectivity between pods that have additional networks that are on different nodes, disable port security for the IPv6 port of the server. Disabling port security obviates the need to create allowed address pairs unnecessary.
```

Reply:
Yes! I again learned a new word.